### ISSUE TITLE
[Issue]: 

### ISSUE THREAD

**Issue by Wavekunai** [Santroller/Santroller#208]
### Please describe your issue.

configuration fails at 18% 

### Version

10.0.19045.0

### What operating system are you using?

Windows 10

### What microcontroller are you using?

pi pico

### How is your controller programmed?

Direct

**Comment by sanjay900** [comment]
Need more info than that, how does it fail

**Comment by Wavekunai** [comment]
yeah sorry about that, i assumed the error would be copied over automatically when i click the report button. i fix the issue. went on the discord server and looked at past issues. i made a new user acount Name just test. and it worked right away.
